logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/UserEventReceiver.kt

```diff
@@ -117,10 +113,6 @@ internal class UserEventReceiverImpl internal constructor(
                             delay = if (event.live) 3.seconds else ZERO
                         )
                         Either.Right(Unit)
-                    }.flatMap {
```

Comment:
issue: `AcceptConnectionRequestUseCase` is executed only when user accepts manually the connection request on current device, so if the "connection accepted" event comes from another device then this warning message won't be added to the conversation